conda_forge_tick/mamba_solver.py

```diff
@@ -297,8 +364,114 @@ def solve(self, specs) -> Tuple[bool, List[str]]:
                 solver.problems_to_str(),
             )
             err = solver.problems_to_str()
+            solution = None
+            run_exports = copy.deepcopy(DEFAULT_RUN_EXPORTS)
+        else:
+            t = api.Transaction(solver, PACKAGE_CACHE)
+            solution = []
+            _, to_link, _ = t.to_conda()
+            for _, _, jdata in to_link:
+                data = json.loads(jdata)
+                solution.append(
+                    " ".join([data["name"], data["version"], data["build"]]),
+                )
+
+            if get_run_exports:
+                run_exports = self._get_run_exports(to_link)
+
+        if get_run_exports:
+            return success, err, solution, run_exports
+        else:
+            return success, err, solution
+
+    def _get_run_exports(self, link_tuples):
+        """Given tuples of (channel, file, json repodata shard) produce a
+        dict with the weak and strong run exports for the packages.
+        """
 
-        return success, err
+        with ProcessPoolExecutor(max_workers=None) as exe:
```

Comment:
Is this something we're going to want to control the max_workers on?

Reply:
Also this seems to not be doing too much heavy lifting in terms of code. Should this be on a thread? Or are we running into "conda isn't threadsafe" issues?

Reply:
We have to decompress the downloaded package which is CPU limited, thus processes seem more appropriate. 

Also, conda on threads scares me too. 

I can add a max_workers constructor argument to the class, but I'm ok letting the system choose.